<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewEditTests</class>
 <widget class="QDialog" name="ViewEditTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="testList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Tests</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textView">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>491</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="calibrationsLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Calibrations: ##</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)</string>
   </property>
   <property name="text">
    <string>Delete File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(213, 101, 52)
</string>
   </property>
   <property name="text">
    <string>Colored</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>text is filled in automatically.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>All spaces are shown as underscores.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ViewEditTests</receiver>
   <slot>onBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ViewEditTests</receiver>
   <slot>onDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>754</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>testList</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ViewEditTests</receiver>
   <slot>on_testChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBack()</slot>
  <slot>onDelete()</slot>
  <slot>on_testChange(QString)</slot>
 </slots>
</ui>
